omnibus/files/private-chef-cookbooks/private-chef/templates/default/openssl.cnf.erb

```diff
@@ -0,0 +1,361 @@
+#
+# OpenSSL example configuration file.
```

Comment:
You are correct, the rest is the default file openssl puts there. Let me see if there is a way to include other files. Might be able to make it better